exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -495,4 +509,50 @@ private void generateOptionsDescriptionJSFile() throws IOException {
       writer.flush();
     }
   }
+
+  //Generates ACE library javascript populated with list of available SQL functions
+  private void generateFunctionJS() throws IOException {
+    //Naturally ordered set of function names
+    TreeSet<String> functionSet = new TreeSet<>();
+    //Extracting ONLY builtIn functions (i.e those already available)
+    List<FunctionHolder> builtInFuncHolderList = this.drillbit.getContext().getFunctionImplementationRegistry().getLocalFunctionRegistry()
+        .getAllJarsWithFunctionsHolders().get(LocalFunctionRegistry.BUILT_IN);
+
+    //Build List of usable functions
```

Comment:
Please explain what functions are considered to be "usable".

Reply:
The ACE library cannot support autocomplete for functions like '<' , '!' and '==' . We filter out all functions that don't start with an alphabet to avoid breaking the suggestions that the library will provide after loading. Added this in the comments now.